prep-javascript-functions/main.js

```diff
@@ -0,0 +1,68 @@
+function addTwoNumbers(num1, num2){
+
+  return num1 + num2;
+}
+const addTwoNumbersResult = addTwoNumbers(6, 6);
+console.log(addTwoNumbersResult);
+
+
+function convertHoursToMinutes(hours){
+const minutes = hours* 60;
+return minutes;
+}
+const convertHoursToMinutesResults = convertHoursToMinutes(19);
+console.log(convertHoursToMinutesResults);
+
+
+function getGreeting(name){
+const hello = 'Hi' + ' ' + 'there';
+return hello;
+}
+const getGreetingResult = getGreeting(name);
+console.log(getGreetingResult);
+
+
+
+function addAndMultiplyBy5(num1, num2){
+return (num1 + num2) * 5;
+}
+const addAndMultiplyBy5Result = addAndMultiplyBy5(310, 424);
+console.log(addAndMultiplyBy5Result);
+
+
+function multiplyAndDivideBy5(num1, num2){
+return (num1 * num2) / 5;
+}
+const multiplyAndDivideBy5Result = multiplyAndDivideBy5(310, 424);
+console.log(multiplyAndDivideBy5Result);
+
+
+function subtractTwoNumbers(num1, num2){
+  return num1 - num2
+}
+const subtractTwoNumbersResult = subtractTwoNumbers(310, 424)
+console.log(subtractTwoNumbersResult)
+
+
+function getCircleCircumfrence(radius){
+  const circumfrence = radius* (2 * 3.14);
+  return circumfrence;
+}
+const getCircleCircumfrenceResult = getCircleCircumfrence(5);
+console.log(getCircleCircumfrenceResult);
+
+
+function getFullName(firstName, lastName){
+const fullName = firstName + lastName;
+  return fullName;
+}
+var fullName = 'Nathaniel Nobleza';
+console.log(fullName);
```

Comment:
You aren't actually calling `getFullName` here. Your `fullName` variable outside the function, should be the result of calling `getFullName` and passing in your name as arguments.